phpcs.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<!-- @see https://pear.php.net/manual/en/package.php.php-codesniffer.annotated-ruleset.php -->
+<ruleset name= "PHPCS Rules">
+  <description>PHPCS ruleset for the DynamoDB ODM</description>
+
+  <file>src</file>
+
+  <!-- Show progress of the run -->
+  <arg value= "p"/>
+
```

Comment:
Can we also explicitly include PSR12 standard here? Like the following:

```xml
<rule ref="PSR12"/>
```

Reply:
Yep, it should be there